src/components/CmsContent.ts

```diff
@@ -35,7 +35,7 @@ export default class CmsContent extends Vue {
 
     return h(dynamic, {
       props: { context: this.context, zoneId: this.zoneId },
-    });
+    }, this.$slots.default);
```

Comment:
it does not seem like you need a slot here. What does passing this here help you accomplish?

Reply:
We went the pop up an alert route. This was the thought that i need to add the id's text next to the content. Will remove this.